uwuno/src/main/java/com/learning/uwuno/cards/deck.java

```diff
@@ -10,15 +10,29 @@
     private LinkedList<card> activeDeck;
     private ArrayList<card> discardPile;
     final private boolean useBlankCards;
-    final private int maxHandSize;
     private card lastCardPlayed;
 
     // Class Functions
-    public deck(boolean useBlankCards, int maxHandSize) {
+    public deck(boolean useBlankCards) {
         activeDeck = createDeck();
         discardPile = new ArrayList<card>();
         this.useBlankCards = useBlankCards;
-        this.maxHandSize = maxHandSize;
+    }
+
+    public LinkedList<card> getActiveDeck() {
+        return activeDeck;
+    }
+
+    public ArrayList<card> getDiscardPile() {
```

Comment:
What is this function used/planned for?

Reply:
We might need it in the future. It also shows up on the JSON response if we need to debug. Otherwise we don't need right now and I can delete it.

Reply:
No that's fine, I just wanted to know